test.py

```diff
@@ -154,9 +154,10 @@ async def get_agent_response(query):
         # To run async code in Streamlit, use asyncio.run or nest_asyncio.
         response = asyncio.run(get_agent_response(user_input))
         with response_container:
-            st.markdown(response)
+print("Line 1 updated")

```

Comment:
Dòng này đang in ra chế độ debug, nên đổi thành production:
```suggestion
print("Production mode")
```